test.sh

```diff
@@ -22,7 +23,7 @@ cleanup || true
 docker build -t $IMAGE  .
 docker run -d --name $PREFIX-db -e POSTGRES_PASSWORD=postgres postgres:10
 docker run -d --name $PREFIX-server --link $PREFIX-db:db \
-    -p 4063:4063 -p 4064:4064 \
+    -p 4063 -p 4064 \
```

Comment:
If 4063 isn't forwarded to 4063 you might as well drop it since you can't pass the insecure port number to a client

Reply:
```suggestion
    -p 4064 \
```